pygeos/constructive.py

```diff
@@ -51,8 +51,8 @@ def boundary(geometry, **kwargs):
     <pygeos.Geometry MULTIPOINT EMPTY>
     >>> boundary(Geometry("POLYGON((0 0, 1 0, 1 1, 0 1, 0 0))"))
     <pygeos.Geometry LINESTRING (0 0, 1 0, 1 1, 0 1, 0 0)>
-    >>> boundary(Geometry("MULTIPOINT (0 0, 1 2)"))
-    <pygeos.NaG>
+    >>> boundary(Geometry("MULTIPOINT (0 0, 1 2)")) is None
```

Comment:
Alternative could be to do `print(..)` which then actually shows `None` (but both are of course workarounds for None having no repr)